src/react/useDiplodocHtml.ts

```diff
@@ -36,9 +36,25 @@ export function useDiplodocEmbeddedContentController(): EmbeddedContentRootContr
 export function useDiplodocEmbeddedContent(embedsConfig?: EmbedsConfig) {
     const controller = useDiplodocEmbeddedContentController();
 
+    // This effect is to clean up all traces of potential automatic initialization (which happens
+    // on DCL), since not doing this would keep everything that was initialized with the old EmbedsConfig
+    // (that is potentially different from default options).
+    // Better solution would be to split extension runtimes: one for dynamic SPA-like scenarios,
+    // where initializing the embed controllers multiple times during the app's lifecycle might be necessary
+    // and one for static content generation, where the initialization is done automatically on DCL.
+    useEffect(() => {
+        controller?.disposeChildren();
+    }, [controller]);
+
+    // Note the deliberate lack of deps. This means that the controllers would get (re)initialized
+    // every render. This works fine in Diplodoc client, since every single re-render of the component hosting the runtime
+    // generally means a navigation was performed.
+    // This is not robust, since extra re-renders would re-initialize all iframes, etc.
+    // However, to keep the runtimes more or less consistent (mermaid-extension, for instance, uses the same
+    // approach), I kept the reliance on this peculiarity for now.
```

Comment:
nit:
I suggest using the pronoun "we" or a call without a pronoun such as "let's continue to use".